discord-scripts/fix-linear-embed.ts

```diff
@@ -12,7 +12,23 @@ import { LinearClient } from "@linear/sdk"
 const { LINEAR_API_TOKEN } = process.env
 
 // track processed message to avoid duplicates if original message is edited
-const processedMessages = new Map<string, Set<string>>()
+const processedMessages = new Map<
+  string,
+  Map<string, { issueId: string; commentId?: string; teamName?: string }>
+>()
+
+// let us also track sent embeds to delete them if the original message is deleted or edited WIP
+const sentEmbeds = new Map<string, Message>()
+
+let issueTagRegex: RegExp | null = null
+
+function initializeIssueTagRegex() {
+  issueTagRegex =
+    /(?<!https:\/\/linear\.app\/[a-zA-Z0-9-]+\/issue\/)[A-Z]{3,}-\d+\b/gi
```

Comment:
At this point we can just put this in a `const` rather than initializing, right?